Library/Homebrew/utils/github.rb

```diff
@@ -123,16 +123,18 @@ def api_credentials_error_message(response_headers, needed_scopes)
             Your macOS keychain GitHub credentials do not have sufficient scope!
             Scopes they need: #{needed_human_scopes}
             Scopes they have: #{credentials_scopes}
-            Create a personal access token: #{ALL_SCOPES_URL}
-            and then set HOMEBREW_GITHUB_API_TOKEN as the authentication method instead.
+            Create a personal access token:
+             #{ALL_SCOPES_URL}
+            #{Utils::Shell.set_variable_in_profile("HOMEBREW_GITHUB_API_TOKEN", "YOUR_TOKEN")}
```

Comment:
The indentation should be 2 spaces everywhere.

Reply:
Fixed!